vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -215,7 +214,7 @@ public void saveElementsNumberByJsonPath(String jsonPath, Set<VariableScope> sco
      * <ul>
      * <li>Executes HTTP GET request to the specified resource</li>
      * <li>Checks if response body contains an element by JSON path</li>
-     * <li>Sleeps during one tenth part of specified duration</li>
+     * <li>Sleeps during during calculated part of specified duration</li>
```

Comment:
shouldn't it go as separate commit?

Reply:
I have updates on this step. So I guess javadoc in scope.

Reply:
But this change has no relation to the step update :)